universal-application-tool-0.0.1/app/views/admin/questions/QuestionEditView.java

```diff
@@ -309,51 +308,64 @@ private DomContent formQuestionTypeSelect(QuestionType selectedType) {
         optionsBuilder.build(),
         questionForm.getEnumeratorId().map(String::valueOf).orElse(NO_ENUMERATOR_ID_STRING));
   }
 
   /**
-   * Generate a {@link SelectWithLabel} enumeration selector with one value from an enumeration
-   * question definition if available, or with just the no-enumeration option.
+   * Generate a {@link SelectWithLabel} enumerator selector with one value from an enumerator
+   * question definition if available, or with just the no-enumerator option.
    */
-  private SelectWithLabel enumerationOptionsFromMaybeEnumerationQuestionDefinition(
+  private SelectWithLabel enumeratorOptionsFromMaybeEnumerationQuestionDefinition(
       Optional<QuestionDefinition> maybeEnumerationQuestionDefinition) {
-    SimpleEntry<String, String> enumerationOption =
+    SimpleEntry<String, String> enumeratorOption =
         maybeEnumerationQuestionDefinition
             .map(
-                enumerationQuestionDefinition ->
+                enumeratorQuestionDefinition ->
                     new SimpleEntry<>(
-                        enumerationQuestionDefinition.getName(),
-                        String.valueOf(enumerationQuestionDefinition.getId())))
+                        enumeratorQuestionDefinition.getName(),
+                        String.valueOf(enumeratorQuestionDefinition.getId())))
             .orElse(new SimpleEntry<>(NO_ENUMERATOR_DISPLAY_STRING, NO_ENUMERATOR_ID_STRING));
-    return enumerationOptions(ImmutableList.of(enumerationOption), enumerationOption.getValue());
+    return enumeratorOptions(ImmutableList.of(enumeratorOption), enumeratorOption.getValue());
   }
 
-  private SelectWithLabel enumerationOptions(
+  private SelectWithLabel enumeratorOptions(
       ImmutableList<SimpleEntry<String, String>> options, String selected) {
     return new SelectWithLabel()
-        .setId("question-enumeration-select")
+        .setId("question-enumerator-select")
         .setFieldName("enumeratorId")
-        .setLabelText("Question enumeration")
+        .setLabelText("Question enumerator")
         .setOptions(options)
         .setValue(selected);
   }
+
+  private ContainerTag repeatedQuestionInformation() {
+    return div("By selecting an enumerator, you are creating a repeated question - a question that"
+                   + " is asked for each repeated entity enumerated by the applicant. Please"
+                   + " reference the applicant-defined repeated entity name to give the applicant"
+                   + " context on which repeated entity they are answering the question for by"
+                   + " using \"$this\" in the question text and help text. To reference the"
+                   + " repeated entities containing this one, use \"$this.parent\","
+                   + " \"this.parent.parent\", etc.")
+        .withId("repeated-question-information")
+        .withClasses(
+            Styles.HIDDEN, Styles.TEXT_RED_500, Styles.TEXT_SM, Styles.P_2, Styles.FONT_MONO);
```

Comment:
Suggestion (blocking): since this isn't an error message, recommend against using red. I think most users associate red text with the app telling them they've done something wrong.